docker/drupal/versions.yaml

```diff
@@ -51,3 +51,22 @@ versions:
   - 9.1-debian9
   - 9.1.11
   - '9.1'
+- dir: 8/php7/debian9/8.9/apache
```

Comment:
Can you please remove 8.9 version? 

I thought we had this version published.

Reply:
version 8.9 was removed